src/nft/components/collection/Card.css.ts

```diff
@@ -24,6 +23,16 @@ export const card = style([
       borderRadius: '16px',
       borderColor: '#5D678524',
       pointerEvents: 'none',
+      zIndex: '2',
+    },
+  },
+])
+
+export const cardHovered = style([
```

Comment:
I think the overall conclusion was that going forward all (even tiny) changes to existing CSS should result in a component being rewritten to Styled Components entirely. 

CC: @cbachmeier